OtripleS.Web.Api.Tests.Unit/Services/StudentGuardianServiceTests/StudentGuardianServiceTests.Exceptions.Delete.cs

```diff
@@ -0,0 +1,173 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using Moq;
+using OtripleS.Web.Api.Models.StudentGuardians;
+using OtripleS.Web.Api.Models.StudentGuardians.Exceptions;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace OtripleS.Web.Api.Tests.Unit.Services.StudentGuardianServiceTests
+{
+   public partial class StudentGuardianServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowDependencyExceptionOnDeleteWhenSqlExceptionOccursAndLogItAsync()
+        {
+            // given
+            Guid randomStudentGuardianId = Guid.NewGuid();
+            Guid inputStudentGuardianId = randomStudentGuardianId;
+            Guid randomStudentId = Guid.NewGuid();
+            Guid inputStudentId = randomStudentId;
+            SqlException sqlException = GetSqlException();
+
+            var expectedStudentGuardianDependencyException
+                = new StudentGuardianDependencyException(sqlException);
```

Comment:
Move the equal sign above

Reply:
Done

Reply:
Mr. @hassanhabib can you please review & merge this action of service, as I am waiting for this.